hibernate-orm-modules/hibernate-orm-modules.gradle

```diff
@@ -108,8 +103,8 @@ provision {
 }
 
 dependencies {
-	testCompile project( ":hibernate-core" )
-	testCompile project( ":hibernate-envers" )
+	compile project( ":hibernate-core" )
```

Comment:
This, and moving test files to `src/main`, looks wrong. What was the problem with keeping the test classes in the test directory?

Reply:
I may be wrong but it seems the two classes were treated as test classes.
`java.lang.Exception: Test class should have exactly one public zero-argument constructor` moving to the `src` folder seems fixing the issue

Reply:
Looks like a gradle configuration problem to me. In Maven, we can set a glob pattern to select which classes should be considered as tests. I suppose you can do the same in Gradle, but I'm curious why the default didn't work, since it works fine for other modules...

Did the failure occur when you explicitly filtered tests by pattern, by any chance? I the pattern was too inclusive, that could explain why some non-test classes were included.

Reply:
not sure if it a gradle or an arquillian problem and what it is even more strange to me is that it depends on the JDK version used